### ISSUE TITLE
Could Not USE Enquiry API When forename is greater then 12 char

### ISSUE THREAD

**Issue by sanjayisg** [reapit/foundations#7598]
Hi I got another Issue, we could add our data with api the first name is having more then 12 char:

[errors] => Array
        (
            [0] => stdClass Object
                (
                    [field] => forename
                    [message] => The value provided exceeds the maximum length of 12 characters
                )

        )

We have many applicant data which have big name grater then 12 char, for example one user is 'Tarlochansingh'

Please remove this validation soon from api.

**Comment by plittlewood-rpt** [comment]
Hi @sanjayisg - this is a limitation with the AgencyCloud database and would need to be dealt with there first before we can alter the rules on the APIs.

**Comment by RuslanButko** [comment]
Hi guys, any updates about this issue?

I have the same situation:

Request Data:
{
enquiryType: salesProperty, 
forename: Chinthiyaruth, 
...

Response:
{dateTime: 2022-09-27T14:50:00.814592Z, 
description: One or more validation failures have occurred. Please refer to Errors list for details, 
errors: [{
field: forename, 
message: The value provided exceeds the maximum length of 12 characters
}], 
statusCode: 422}

**Comment by HollyJoyPhillips** [comment]
Internal reference ENG-82
